### DISCUSSION TITLE
What is the purpose of the array splice method in JavaScript?

### DISCUSSION THREAD

**Discussion by montasim** [STRINGLABITSOLUTIONS/Bhojon#344] [Q&A] [answered]
In short.

**Comment by d4tdev** [comment, marked as answer]
The splice() method changes the contents of an array by removing or replacing existing elements and/or adding new elements in place

See details at: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice#:~:text=The%20splice()%20method%20changes,adding%20new%20elements%20in%20place.

**Comment by Preetiraj3697** [comment]
The slice() method returns selected elements in an array, as a new array.
The slice() method selects from a given start, up to a (not inclusive) given end.
The slice() method does not change the original array.
